### PR TITLE
Climate profile unittest

### PR DESCRIPTION
**Pull request by Desipeli**

### CODE REVIEW

File: frontend/cypress/e2e/climateProfiles.cy.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+describe('Climate profiles page', function () {
+    beforeEach(function () {
+        cy.visit('http://localhost:5173/profiles')
```
Comment (PaulusParssinen) [inline frontend/cypress/e2e/climateProfiles.cy.js:3]: Is `http://localhost:5173` prefix required or can it be relative like: `/profiles`?

**Review by leinson — CHANGES_REQUESTED** [review]
add for smaller viewport?

File: frontend/cypress/e2e/climateProfiles.cy.js (hunk 1 above)
Comment (Desipeli) [reply frontend/cypress/e2e/climateProfiles.cy.js]: Now it can be relative. I added http://localhost:5173 to cypress.config.js

Comment (PaulusParssinen) [reply frontend/cypress/e2e/climateProfiles.cy.js]: That'll work, will have to look at why Vite doesn't proxy these e2e requests.

**Review by leinson — APPROVED** [review]
good changes, all good now

**Pull request merged**